mindmeld/converter/dialogflow.py

```diff
@@ -182,22 +186,24 @@ def _create_intents_directories(self, intents):
                 mindmeld_intent_directory_name = self.clean_check(
                     sub, self.intents_list
                 )
+
+                domain = 'unrelated' if \
+                    'default' in mindmeld_intent_directory_name else 'app_specific'
```

Comment:
if it has to split the line, we should just use a regular `if` block

Reply:
done